components/org.wso2.carbon.identity.scim.common/src/main/java/org/wso2/carbon/identity/scim/common/internal/SCIMCommonComponent.java

```diff
@@ -88,4 +100,27 @@ protected void setIdentityCoreInitializedEventService(IdentityCoreInitializedEve
          is started */
     }
 
+    /**
+     * Set realm service implementation
+     *
+     * @param realmService RealmService
+     */
+    protected void setRealmService(RealmService realmService) {
+
+        if (logger.isDebugEnabled()) {
+            logger.debug("realmService set in SCIMCommonComponent bundle");
+        }
+        SCIMCommonComponentHolder.setRealmService(realmService);
+    }
+
+    /**
+     * Unset realm service implementation
+     */
+    protected void unsetRealmService(RealmService realmService) {
+
+        if (logger.isDebugEnabled()) {
+            logger.debug("realmService unset in SCIMCommonComponent bundle");
```

Comment:
Start the log message with a Capital letter and end with a full stop.

Reply:
Fixed with #73